Makefile

```diff
@@ -44,13 +44,25 @@ my.env:
 .docker-build:
 	make build
 
+.devcontainer-build:
+	make devcontainerbuild
+
 .PHONY: build
 build: my.env  ## | Build docker images.
 	${DC} build ${DOCKER_BUILD_OPTS} --build-arg userid=${SOCORRO_UID} --build-arg groupid=${SOCORRO_GID} --progress plain app
 	${DC} build --progress plain oidcprovider fakesentry
 	${DC} build --progress plain statsd postgresql memcached localstack elasticsearch symbolsserver
 	touch .docker-build
 
+.PHONY: devcontainerbuild
+devcontainerbuild: my.env  ## | Build VS Code development container.
```

Comment:
Aside: I noticed the other Crash Ingestion repos just use `.env`, but Socorro uses `my.env`. Doesn't matter really as long as your filename matches the filename it's looking for. Just a minor inconsistency across the repos.

Reply:
i think it's worth standardizing on `.env` because that also automatically pulled in by docker compose, but definitely a concern for a separate bug. I'll file one and link it here shortly.

Reply:
Tecken uses `.env` because that's what peterbe did. Eliot uses `.env` because I extracted it out of Tecken.

Socorro and Antenna use `my.env` instead because I don't like using `.env` for service things because so many other things (docker-compose, honcho, etc) use `.env` that it sometimes creates problems. Plus `.env` is a dotfile so it doesn't show up in listings.

Having said that, we should probably standardize on `.env` and suffer whatever issues come up.

Reply:
filed https://bugzilla.mozilla.org/show_bug.cgi?id=1884446 about changing to `.env` here and in antenna